wat niet doen: 

} catch (Exception e) {
				// TODO Auto-generated catch block
				e.printStackTrace();
			}

wel: ISOException.throwIt(ERROR_UNKNOW);
die eerste screenshot is goed! gwn (short) (bewerking)

dit is ok: byte [] responseChallengeBytes = new byte[16];